src/manifolds/GraphManifold.jl

```diff
@@ -210,28 +224,11 @@ function manifold_dimension(M::GraphManifold{<:AbstractGraph,<:Manifold,EdgeMani
     return manifold_dimension(M.manifold) * ne(M.graph)
 end
 
-@doc doc"""
-    representation_size(N::GraphManifold{G,M,VertexManifold})
-
-returns the representation size of a point on the [`GraphManifold`](@ref) on the vertices of $G=(V,E),
-which is the representation size of `M` with $\lvert V \rvert$ added.
-"""
-function representation_size(M::GraphManifold{<:AbstractGraph,<:Manifold,VertexManifold})
-    return (representation_size(M.manifold)..., nv(M.graph))
-end
-@doc doc"""
-    representation_size(N::GraphManifold{G,M,EdgeManifold})
-
-returns the representation size of a point on the [`GraphManifold`](@ref) on the vertices of $G=(V,E),
-which is the representation size of `M` with $\lvert E \rvert$ added.
-"""
-function representation_size(M::GraphManifold{<:AbstractGraph,<:Manifold,EdgeManifold})
-    return (representation_size(M.manifold)..., ne(M.graph))
-end
-
+#=
```

Comment:
what happened to the representation size – is it now just the one from `Power`?

Reply:
There is none, essentially. If something isn't a flat array, it doesn't have representation size. We can think about returning `nothing` by default in such cases instead of throwing an error.